packages/server/src/sdk/app/rowActions.ts

```diff
@@ -39,15 +45,53 @@ export async function create(tableId: string, rowAction: { name: string }) {
     doc = { _id: rowActionsId, actions: {} }
   }
 
+  const { name: tableName } = await tables.getTable(tableId)
+
   ensureUniqueAndThrow(doc, action.name)
 
+  const automation = await automations.create({
+    name: `${tableName}: ${action.name}`,
+    appId: context.getAppId()!,
```

Comment:
Would it be better to instead check that `context.getAppId()` is defined and throw an error if not? The `!` is making me nervous, don't want to store bad data.